area4/__init__.py

```diff
@@ -65,7 +65,16 @@ def divider(self, number) -> str:
                 return self.dividers[number].split("\n")[0]
             except IndexError:
                 raise ValueError('That divider doesn\'t exist!')
-
+    def logDivider(self, div, *stuff) -> None:
```

Comment:
Needs DocStrings/PyDoc

Reply:
Could you add that? I'm not sure if I could do it right.

Reply:
Sure